yarn.lock

```diff
@@ -0,0 +1,4918 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
It contains the [dependency tree](https://yarnpkg.com/en/docs/yarn-lock#toc-check-into-source-control).